AsyncDisplayKit/TextKit/ASTextKitRenderer.mm

```diff
@@ -299,6 +299,16 @@ - (NSUInteger)lineCount
   return lineCount;
 }
 
+- (BOOL)isTruncated
+{
+  if (self.canUseFastPath) {
+    CGRect boundedRect = [_attributes.attributedString boundingRectWithSize:CGSizeMake(_constrainedSize.width, CGFLOAT_MAX) options:NSStringDrawingUsesLineFragmentOrigin | NSStringDrawingTruncatesLastVisibleLine context:nil];
+    return boundedRect.size.height > _constrainedSize.height;
```

Comment:
It looks like this check is safe even if attributedString is nil - a CGRectZero will be returned and this should always pass as "nope, not truncated" even if constrainedSize.height is zero.